### ISSUE TITLE
Cannot create backup

### ISSUE THREAD

**Issue by dknx01** [andOTP/andOTP#589]
#### General information

* **App version:*  0.7.1.1-play*
* **App source:** Google Play
* **Android Version:** 10 (Patchlevel: 05. April 2020)
* **Custom ROM:**

#### Expected result
**Create a backup file (plain text) for all my keys** 

**An error occurred that the software cannot create the backup directory.
If I set one manually in the settings, the software cannot write to it.**

Related to issue #530 (seems still be a problem)

#### Logcat
```
06-16 08:34:17.402 28593 28593 I Surface : opservice is null false
06-16 08:34:17.403   826  1457 E SurfaceFlinger: Permission Denial: SurfaceFlinger did not recognize request code: 20001
06-16 08:34:17.403  2448  2528 E OnePlusService: Exception notifySurfaceFlingerUpdate
06-16 08:34:17.404  2448  2528 W System.err: java.lang.SecurityException
06-16 08:34:17.404  2448  2528 W System.err:    at android.os.BinderProxy.transactNative(Native Method)
06-16 08:34:17.404  2448  2528 W System.err:    at android.os.BinderProxy.transact(BinderProxy.java:557)
06-16 08:34:17.404  2448  2528 W System.err:    at com.oneplus.server.OnePlusService.notifySurfaceFlingerUpdate(Unknown Source:21)
06-16 08:34:17.404  2448  2528 W System.err:    at com.oneplus.os.IOnePlusService$Stub.onTransact(IOnePlusService.java:347)
06-16 08:34:17.404  2448  2528 W System.err:    at android.os.Binder.execTransactInternal(Binder.java:1032)
06-16 08:34:17.404  2448  2528 W System.err:    at android.os.Binder.execTransact(Binder.java:1005)

```

#### Steps to reproduce
 
 * open the app
 * try to create a plain text backup

**Comment by flocke** [comment]
This should be fixed in the upcoming 0.8.0 release, which reworks the entire storage handling.

**Comment by dknx01** [comment]
Ah thanks for the information. I'll wait

**Comment by flocke** [comment]
Should be fixed in 0.8.0, just reopen if not.
